CHANGELOG.md

```diff
@@ -5,6 +5,13 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
+## [v6.3.0](https://github.com/eduNEXT/eox-hooks/compare/v6.2.0...v6.3.0) - (2024-07-24)
+
+### Changed
+
+- **Redwood Support**: Upgrade requirements base on edx-platform redwood
```

Comment:
Could we add an entry for the integration tests and another for Redwood support in the changelog? You can follow the format used in this example: https://github.com/eduNEXT/eox-tenant/blob/master/CHANGELOG.md?plain=1#L8-L20

Reply:
@magajh Thanks for the feedback